api/clients/eigenda_client_e2e_test.go

```diff
@@ -22,9 +22,9 @@ func init() {
 // We don't test waiting for finality because that adds 12 minutes to the test, and is not necessary
 // because we already test for this in the unit tests using a mock disperser which is much faster.
 func TestClientUsingTestnet(t *testing.T) {
-	if !runTestnetIntegrationTests {
-		t.Skip("Skipping testnet integration test")
-	}
+	//if !runTestnetIntegrationTests { // TODO
+	//}
+	t.Skip("Skipping testnet integration test")
```

Comment:
This test is currently failing on multiple different branches.

Reply:
This test randomly started working for me on a different branch, I wonder if testnet was just having trouble